src/mame/sinclair/scorpion.cpp

```diff
@@ -520,6 +614,7 @@ void scorpiontb_state::machine_reset()
 
 	scorpion_state::machine_reset();
 	m_is_m1_even = 0;
+	m_io_turbo->field(0x01)->live().value = 0;
 	m_maincpu->set_clock_scale(1);
```

Comment:
You really shouldn’t be messing with the value of toggle controls like this – it’s by design that they “stick” across runs.  You switch the machine off, the toggle switch stays in the same position.